packages/state-transition/src/block/externalData.ts

```diff
@@ -13,13 +13,14 @@ export enum ExecutionPayloadStatus {
   valid = "valid",
 }
 
-export enum DataAvailableStatus {
-  preDeneb = "preDeneb",
-  notAvailable = "notAvailable",
```

Comment:
I double checked and this condition/enum value was not used anywhere but `outOfRange` is used by the validator api

Reply:
will check and get back on this